project/npda/models/npda_user.py

```diff
@@ -104,37 +96,29 @@ def create_superuser(self, email, password, **extra_fields):
                 logged_in_user = self.create_user(
                     email.lower(), password, **extra_fields
                 )
-                paediatric_diabetes_unit, created = (
-                    PaediatricDiabetesUnit.objects.get_or_create(
-                        pz_code="PZ999",
-                        ods_code="8HV48",
-                    )
+                paediatric_diabetes_unit = PaediatricDiabetesUnit.objects.get(
+                    pz_code="PZ999",  # RCPCH
                 )
-                organisation_employer, created = (
-                    OrganisationEmployer.objects.get_or_create(
-                        is_primary_employer=True,
-                        paediatric_diabetes_unit=paediatric_diabetes_unit,
-                        npda_user=logged_in_user,
-                    )
+                # if user already has an employer, do not create a new one - update the status to primary
+                OrganisationEmployer.objects.update_or_create(
+                    paediatric_diabetes_unit=paediatric_diabetes_unit,
+                    npda_user=logged_in_user,
+                    defaults={"is_primary_employer": True},
                 )
             else:
                 extra_fields.setdefault("is_rcpch_staff", False)
                 extra_fields.setdefault("view_preference", 2)  # national scope
                 logged_in_user = self.create_user(
                     email.lower(), password, **extra_fields
                 )
-                paediatric_diabetes_unit, created = (
-                    PaediatricDiabetesUnit.objects.get_or_create(
-                        ods_code="RJZ01",
-                        pz_code="PZ215",
-                    )
+                paediatric_diabetes_unit = PaediatricDiabetesUnit.objects.get(
+                    pz_code="PZ215",  # Superusers that are not RCPCH staff are affiliated with King's College Hospital
```

Comment:
Would we have any superusers not attached to a PDU or the RCPCH?

Reply:
no. I don't think so. I the group 4 are rcpch employees only. clinicians can have rcpch permissions but would have the is_rcpch_audit_team_member as true, unless you want us to remove that.